### ISSUE TITLE
Separate feedback questions for different adj types

### ISSUE THREAD

**Issue by philipbelesky** [TabbycatDebate/tabbycat#909]
As discussed, this separation might be quite light and not involve fundamental changes to the model. Instead, differentiation could be largely handled on the front-end.

**Comment by tienne-B** [comment]
Please elaborate! :) I also don't see how this could be done without adding a field to the question table.

**Comment by philipbelesky** [comment]
Yes, there will definitely be at least a field addition. @czlee is probably better placed to discuss implementations — but I think the idea was that the the form logic and submission model doesn't have the capacity to do more advanced filtering based on type.
